File: Projetos/ACBrLib/Demos/C#/NFe/Imports/Dinamico/Shared/NotaFiscal/DadosAdicionaisNFe.cs
using System.Collections.Generic;
using ACBrLib.Core.DFe;

namespace ACBrLib.NFe
{
    /// <summary>
    /// Informações Adicionais da NF-e
    /// </summary>
    public class DadosAdicionaisNFe : DadosAdicionais
    {
        #region Properties

        public List<InfoAdicionalNfe> ObsCont { get; } = new List<InfoAdicionalNfe>();

        public List<InfoAdicionalNfe> ObsFisco { get; } = new List<InfoAdicionalNfe>();

        public List<ProcRefNFe> ProcRef { get; } = new List<ProcRefNFe>();

        #endregion Properties
    }
}

File: Projetos/ACBrLib/Android/NFe/ACBrLibNFe/src/main/java/br/com/acbr/lib/nfe/notafiscal/DadosAdicionaisNFe.java
package br.com.acbr.lib.nfe.notafiscal;

import java.util.ArrayList;
import java.util.List;

public class DadosAdicionaisNFe {
    private final List<InfoAdicionalNfe> obsCont = new ArrayList<>();
    private final List<InfoAdicionalNfe> obsFisco = new ArrayList<>();
    private final List<ProcRefNFe> procRef = new ArrayList<>();

    public List<InfoAdicionalNfe> getObsCont() {
        return obsCont;
    }

    public List<InfoAdicionalNfe> getObsFisco() {
        return obsFisco;
    }

    public List<ProcRefNFe> getProcRef() {
        return procRef;
    }
}
